classes/03class/exercises/c03-docker01/nimmi89/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:latest
```

Comment:
using latest tag is never a good practice as the image can change upstream and affect your build.

Can you use a tag other than `latest?`

Reply:
Yes!Makes sense!! Thanks for that Kiko.Have made the change.